doc/whats_new/v0.20.rst

```diff
@@ -100,3 +109,12 @@ Linear, kernelized and related models
 - Deprecate ``random_state`` parameter in :class:`svm.OneClassSVM` as the
   underlying implementation is not random.
   :issue:`9497` by :user:`Albert Thomas <albertcthomas>`.
+
+Decomposition, manifold learning and clustering
+
+- Deprecate ``precomputed`` parameter in function
+  :func:`manifold.t_sne.trustworthiness`. Instead, the new parameter
+  ``metric`` should be used with any compatible metric including
+  'precomputed', in which case the input matrix ``X`` should be a matrix of
+  pairwise distances or squared distances. :issue:`9736` by
+  :user:`Joel Nothman <jnothman>`.
```

Comment:
This should be your name and the issue number should be 9775

Reply:
@jnothman 9775 is the id of the PR not the issue. Do we refer to the issue or to the PR? [here](https://github.com/scikit-learn/scikit-learn/pull/9059/files#diff-3433b590ca8611f80d7687b51a869383R40) we are referring to the issue. Is there a defined criteria? If so, I think we should add it in the contributing guide [here](http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines) and be part of the reviewing checklist discussed in #9653